src/__tests__/bulkUploads.int.test.ts

```diff
@@ -184,6 +184,46 @@ describe('/bulkUploads', () => {
 				);
 		});
 
+		it('returns uploads for the admin user when createdBy is set to me as an admin', async () => {
```

Comment:
Out of curiosity, why are these tests framed around admins? It implies to me this functionality is meant to only work for admins, when that's not the case. And in fact I think it'd be good to have tests that ensure this *isn't* somehow an admin-only feature. Right?

I'm not insistent on this point, and maybe there's a good reason!

Reply:
Main reason right now is that admin is the only role where the filter matters.  If you aren't an admin you only get your own things anyway.

"admin" in the test name is just placeholder for "user where more than just their stuff would have been returned in the first place"